File: libreria/frontend/libreria_front/lib/models/news.dart
class News {
  final String id;
  final String title;
  final String description;
  final String summary;
  final String image;

  News({
    required this.id,
    required this.title,
    required this.description,
    required this.summary,
    required this.image,
  });

  factory News.fromJson(Map<String, dynamic> json) {
    return News(
      id: json['id'],
      title: json['title'],
      description: json['description'],
      summary: json['summary'],
      image: json['image'],
    );
  }
}

File: libreria/backend/project/bussiness/site/news.py
from sqlalchemy import Column, Integer, String, ForeignKey, Float, Boolean, Table
from sqlalchemy.ext.declarative import declarative_base
from sqlalchemy.orm import relationship

Base = declarative_base()

class News(Base):
    __tablename__ = 'news'
    id = Column(Integer, primary_key=True)
    title = Column(String(50), nullable=False)
    summary = Column(String(500), nullable=False)
    description = Column(String(1500), nullable=False)
    image = Column(String(100), nullable=False)

    def __init__(self, title: str, summary:str, description: str, image: str):
        self.title = title
        self.summary = summary
        self.description = description
        self.image = image